README.md

```diff
@@ -1 +1,122 @@
-# Enumerables
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[Enumerables\] ](#-morse-code-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+# 📖 [Enumerables] <a name="about-project"></a>
+
+ [Enumerables] Is a project that decrypts morse code using ruby
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Tech stack</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Implements_a_subset_of_the_functionality_enumerable]**
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+ - Ruby installed
+
+### Setup
+
+Clone this repository to your desired folder:
+- Open git bash on local computer
+- clone the repo with:
+
+  ```sh
+  git clone https://github.com/emmanuelsteven/Enumerables.git
+  cd Enumerables
+  ```
+
+### Install
+
+Install this project with:
+
+```sh
+bundle install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+- Open the terminal and insert the commands in this repository.
+```
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Emmanuel Steven**
+
+- GitHub: [emmanuelsteven](https://github.com/emmanuelsteven)
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] [Add_a_user_interface_for_the_enumerables]
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [Isues Pages](https://github.com/emmanuelsteven/Enumerables/issues)
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project kindly give it a star⭐️.
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank cpatain Bawa for the immense support and insights.
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/emmanuelsteven/Enumerables/blob/dev/LICENSE) licensed.
```

Comment:
- [ ] Kindly make sure to add a license to your project and check this link it's broken. By fixing all the links in your README your project will appear as well-documented and complete